static/panes/conformance-view.js

```
@@ -419,7 +419,7 @@ Conformance.prototype.compileChild = function (compilerEntry) {
                             asm: '',
                             code: -1,
                             stdout: '',
-                            stderr: x.error,
+                            stderr: x.error || x.message || x,
```

This catch has been eating the exception in the code this whole time